src/test/java/api/loans/RenewByIdTests.java

```diff
@@ -7,8 +7,22 @@
 import org.folio.circulation.support.http.client.Response;
 import org.folio.circulation.support.http.server.ValidationError;
 import org.hamcrest.Matcher;
+import org.junit.After;
+import org.junit.Before;
 
 public class RenewByIdTests extends RenewalAPITests {
+
+  @Before
+  public void setupLoanType() {
+    noteTypeFixture.generalNoteType();
+  }
+
+  @After
+  public void cleanUp() {
+    notesClient.deleteAll();
+    noteTypeClient.deleteAll();
```

Comment:
Given how often this is done, might it be worth setting up the type and doing this clean up in the API tests in general?

Reply:
It only affected about 85 tests, I didn't think that warranted running it for every test. I can move it into the base class if you think it's worth the overhead.

Reply:
Ok, let's leave it for now. That it is needed in so many places suggests that the tests might not be structured well.